README.md

```diff
@@ -29,6 +31,16 @@ Add this to .vimrc:
 Bundle 'git://github.com/miripiruni/CSScomb-for-Vim.git'
 ```
 
+### With NeoBundle
+Add this to .vimrc:
+```
+NeoBundle 'faceleg/vim-csscomb', {
+        \ 'build': {
+        \     'unix': 'npm install -g csscomb',
+        \     'mac': 'npm install -g csscomb'
+        \ }}
```

Comment:
I'm not familiar with NeoBundle. Could you please explain what's happening here?